subsys/mgmt/hawkbit/hawkbit_priv.h

```diff
@@ -14,6 +14,7 @@
 #define __HAWKBIT_PRIV_H__
 
 #include <data/json.h>
+#include "hawkbit_config_data.h"
```

Comment:
I'm not sure if a system application is allowed to include a header from the user application

I'm not sure who to ask though, @gmarull you have any idea about this?